packages/search/src/index.ts

```diff
@@ -1 +1,2 @@
 export { default } from './search';
+export { useFuzzySearch } from './use-fuzzy-search';
```

Comment:
Should we document usage in `packages/search/README.md`?

I think it'd be really nice to have some examples of normal search vs. fuzzy search.

Reply:
Pasting a couple of the unit test examples into the README sounds good.

Reply:
Looking at the `README`, this package is responsible for providing the `<Search />` component and that's it. We have two ways of going here:

1. Modify the `README` so that it's clear that not only the component is exported, but also a hook that allows fuzzy searching. I thought about changing from "A search input component" to "Search utilities" and then include new headings showcasing the component and the hook;
2. Create a new package that only exposes the fuzzy search hook. This way we'd isolate things a bit and the README would be focused on explaining just the fuzzy search functionality.

Which one do you prefer? There are cases for both sides here.

`1` is all about the hook being very closely related to the component domain-wise, so it makes sense to stick it there. However, the `<Search />` component does not perform the search itself, it's just the input. The actual search mechanism is decoupled and totally set up by the client, so it's not like the component is going to use the hook.

So in this case, actually `2` would be better since they're domain-wise related, but the use case for the components diverge. How hard is it to create a new package, and does it make sense? Looking for input here.

cc @tyxla

Reply:
I'm fine with either, TBH. If these can (and need to) be functionally and architecturally decoupled in a proper way, they can totally be separate packages. I can see how fuzzy search is separate, but I can also see it as an attachment to the broader search package.

Reply:
Yep, yep. That makes total sense. I'll go with #1.